client/web/src/search/home/VulnerabilityAnnouncement.tsx

```diff
@@ -8,22 +9,39 @@ import styles from './VulnerabilityAnnouncement.module.scss'
 /**
  * TODO description
```

Comment:
Once again trying to sneak some different description in here 😆 Would love to not have a TODO comment in the codebase for this :) 

```suggestion
 * Displays a banner linking to the log4j blog post.
```

Reply:
Sorry for forgetting about it in my last PR, just updated it, thanks @eseliger !

Reply:
Nice, may be helpful to document that this is a temporary component